### ISSUE TITLE
User Guide Doxygen cutting parts out 

### ISSUE THREAD

**Issue by zanzaben** [nasa/cFE#1183]
**Describe the bug**
There are parts of the User Guide that when formatted by Doxygen are causing parts of the content to get cut out and not included. I happened to find one but imagine there are others. 

**To Reproduce**
The one example I found was on page 317 for 37.30.2.1 CFE_TBL_Register() The TblOptionFlags is missing the last 2 sentences of CFE_TBL_OPT_CRITICAL. 

**Expected behavior**
No information should be lost by Doxygen formatting.

**Code snips**
https://github.com/nasa/cFE/blob/c708d64bf2b25500cd0308a700aa4ee0b3fd5909/fsw/cfe-core/src/inc/cfe_tbl.h#L226-L235

**System observed on:**
 - Adobe Acrobat Reader 2017

**Reporter Info**
Alex Campbell GSFC

**Comment by skliper** [comment]
Good catch.  I wonder if there is a setting in doxygen for how long the description of an arg can be?
